lib/go/test/flow_dkg_test.go

```diff
@@ -19,10 +19,6 @@ import (
 	"github.com/onflow/flow-core-contracts/lib/go/templates"
```

Comment:
These tests are updated to the point where they are passing, however there are still several places (noted as TODO comments) where additional updates should be made prior to merging. In particular cases where the test is expecting a failure, but the failure occurs for the wrong reason (like passing in the old transaction arguments).